File: sdkwork-ai-sdk-go/types/assistant_response.go
package types

// 助手响应
type AssistantResponse struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Model string `json:"model"`
	Name string `json:"name"`
	Description string `json:"description"`
	Instructions string `json:"instructions"`
	Tools []interface{} `json:"tools"`
	Metadata map[string]string `json:"metadata"`
	Temperature float64 `json:"temperature"`
	CreatedAt int `json:"created_at"`
	ToolResources map[string]interface{} `json:"tool_resources"`
	ResponseFormat interface{} `json:"response_format"`
	TopP float64 `json:"top_p"`
}

File: sdkwork-ai-sdk-csharp/Models/AssistantResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class AssistantResponse
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public string? Model { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
        public string? Instructions { get; set; }
        public List<object>? Tools { get; set; }
        public Dictionary<string, string>? Metadata { get; set; }
        public double? Temperature { get; set; }
        public int? CreatedAt { get; set; }
        public Dictionary<string, object>? ToolResources { get; set; }
        public object? ResponseFormat { get; set; }
        public double? TopP { get; set; }
    }
}
